pkg/webui/console/containers/gateway-events/index.js

```diff
@@ -51,6 +51,8 @@ const GatewayEvents = props => {
     dispatch(clearGatewayEventsStream(gtwId))
   }, [dispatch, gtwId])
 
+  const filteredEvents = useMemo(() => events.filter(e => !/gcs\.managed\./.test(e.name)), [events])
```

Comment:
If we introduce a separate event subscription just for managed gateway events, there's a bunch of backend complexity w.r.t. event streams that we can revert. I'm okay with either, I thought it's elegant to piggyback all managed gateway events on the existing stream. So if we're gonna do that, please do so via an issue.

To the point being made here: I think it does make sense to have two levels of event filtering, one at the subscription level and the other one at the display level.

Reply:
Isn't the backend using the same stream connection anyway internally for the same entity? I thought that's how it worked since moving to the websocket implementation, where there is always one connection anyway with a pub/sub interface.

Reply:
Anyway, let's leave it as it is for now.